checkmarx-ast-eclipse-plugin/src/com/checkmarx/eclipse/views/DataProvider.java

```diff
@@ -137,7 +137,8 @@ public void setCurrentResults(Results currentResults) {
 			try {
 				CxWrapper cxWrapper = getWrapper();
 				
-				branchList = cxWrapper.projectBranches(UUID.fromString(projectId), PluginConstants.EMPTY_STRING);
+				if(projectId != null && projectId != "")
```

Comment:
Consider adding brackets even if the "if" contains only one statement as it can lead to mistakes